src/mapbox.android.ts

```diff
@@ -551,7 +551,7 @@ export class Mapbox extends MapboxCommon implements MapboxApi {
 
   private _mapboxMapInstance: any;
   private _mapboxViewInstance: any;
-
+  private _activity: string;
```

Comment:
Could I trouble you to add a comment block here describing its reason for being/the problem it's addressing? This will make it easier for subsequent developers to come up to speed.

Reply:
Done. Is that clear enough?

Reply:
Yes. Thank you.